cmd/goProbe/config/config.go

```diff
@@ -252,6 +251,12 @@ func RuntimeDBPath() string {
 	return runtimeDBPath.path
 }
 
+// SetRuntimeDBPath explicitly sets the DB path
+func SetRuntimeDBPath(path string) {
```

Comment:
Had to make this accessible (ideally this wouldn't be some global, but I didn't really feel like refactoring everything just to provide a clean interface for both goProbe and goQuery).